lib/request.js

```diff
@@ -0,0 +1,45 @@
+'use strict'
+
+var https = require('https')
+var parseUrl = require('url').parse
+
+function post (endpoint, body, PromiseWrapper) {
```

Comment:
Couldn't think of a better way to phrase the promise argument. Alternatively, I could make this a private method on the `SlackWebhook` object and then reference `this.url` and `this.Promise` instead. Up to you.